terraso_backend/apps/graphql/schema/commons.py

```diff
@@ -161,6 +165,11 @@ def not_allowed(cls, mutation_type=None, msg=None, extra=None):
     def not_allowed_create(cls, model, msg=None, extra=None):
         raise cls.not_allowed(MutationTypes.CREATE, msg, extra)
 
+    @classmethod
+    def not_found(cls, model=None, field=None, msg=None):
+        model = model or cls.model_class
+        raise GraphQLNotFoundException(msg, field=field, model_name=model.__name__)
+
```

Comment:
question: can we use/combine with `not_found` from BaseMutation?

Reply:
yes, this is meant to be a super call with the default value for model supplied!